libs/chlorophyll/scss/components/slider/_index.scss

```diff
@@ -0,0 +1,67 @@
+@use 'mixins' as *;
+
+input[type='range'] {
+  -webkit-appearance: none;
+  width: 100%;
+  height: $track-height;
+  background: linear-gradient(
+    to right,
+    $dark-blue-2 0%,
+    $dark-blue-2 0%,
+    $track-color 0%,
+    $track-color 100%
+  );
+  border-radius: $track-border-radius;
+
+  @each $browser in $browser-thumb-classes {
+    &::#{$browser} {
+      @include thumb-styles();
+    }
+  }
+
+  &:focus {
+    outline: none;
+
+    @each $browser in $browser-thumb-classes {
+      &::#{$browser} {
+        @include thumb-focus-styles();
+      }
+    }
+  }
+
+  &:disabled {
+    @each $browser in $browser-thumb-classes {
+      &::#{$browser} {
+        @include thumb-disabled-styles();
+      }
+    }
+  }
+}
+
+.ngg-slider {
```

Comment:
The ngg namespace refers to "Angular Green" and should probably not be used in chlorophyll. I think it should be swapped with "gds", can you confirm @vsjolander ?

Reply:
I think you're right. My bad. Will rename my css classes :D